### PR TITLE
fix(SUP-28319): prevent focus chapters when side panel closed

### PR DESCRIPTION
**Pull request by semarche-kaltura**
1) use css property "visibility: hidden" to prevent tabbing into closed sidebar panel;
2) handle "home"\"end" keys for navigation to first\last chapter(slide)

### CODE REVIEW

File: modules/KalturaSupport/components/chapters/chapters.js [hunk 1]
```diff
@@ -1311,10 +1324,12 @@
...
 			var mediaBoxes = this.getMediaListDomElements();
-			mediaBoxes.on('mousedown mouseup mouseout', function(){
-				this.blur();
-			})
-			.on('keyup', $.proxy(this.onItemKey , this));
+			mediaBoxes
+				.on('mousedown mouseup mouseout', function(){
+					this.blur();
+				})
+				.on('keydown', $.proxy(this.handleHomeEndKeys, this))
```
Comment (OrenMe) [inline modules/KalturaSupport/components/chapters/chapters.js:1331]: why separate keydown and keyup? does it matter if we use any of them? if not lets consolidate to one and reuse `onItemKey` for all

Comment (semarche-kaltura) [reply modules/KalturaSupport/components/chapters/chapters.js:1331]: player handle 'keyup' before plugin and makes seekToBeginning and seekToEnd

Comment (OrenMe) [reply modules/KalturaSupport/components/chapters/chapters.js:1331]: For this case there is `onDisableKeyboardBinding` event to disable the main player keyboard bindings and `onEnableKeyboardBinding` to bring them back.

Comment (RoyBregman) [reply modules/KalturaSupport/components/chapters/chapters.js:1331]: I think for this scenario, Yuval created this:
https://github.com/kaltura/playkit-js-ui/blob/master/docs/with-keyboard-event.md
@OrenMe @yairans ?

Comment (semarche-kaltura) [reply modules/KalturaSupport/components/chapters/chapters.js:1331]: cool,
I'll make the changes with `onDisableKeyboardBinding` and `onEnableKeyboardBinding`

Comment (RoyBregman) [reply modules/KalturaSupport/components/chapters/chapters.js:1331]: As the solution @OrenMe suggested turns out to be much more complex (due to other `onEnableKeyboardBinding` and `onDisableKeyboardBinding` which are fired irrelevantly) we will still stick with @semarche-kaltura solution

**Review by RoyBregman — APPROVED** [review]

**Pull request merged**